src/cli/cli.cpp

```diff
@@ -7754,6 +7754,36 @@ template <> otError Interpreter::Process<Cmd("vendor")>(Arg aArgs[])
         error = ProcessGetSet(aArgs, otThreadGetVendorSwVersion, otThreadSetVendorSwVersion);
 #endif
     }
+    /**
+     * @cli vendor appurl
+     * @code
+     * vendor appurl
+     * http://www.example.com
+     * Done
+     * @endcode
+     * @par api_copy
+     * #otThreadGetVendorAppUrl
+     */
+    else if (aArgs[0] == "appurl")
+    {
+        aArgs++;
+
+#if !OPENTHREAD_CONFIG_NET_DIAG_VENDOR_INFO_SET_API_ENABLE
+        error = ProcessGet(aArgs, otThreadGetVendorAppUrl);
+#else
+        /**
+         * @cli vendor appurl (url)
```

Comment:
```suggestion
         * @cli vendor appurl (set)
```

For these we should use (set) for consistency with other commands unless we feel there will be different versions of setting things.

Can you change these too to have (set) instead, since you're updating this file anyway?  It's just changing the label for each `@cli` line to be (set) instead.

* vendor model (name)
* vendor name (name)
* vendor swversion (version)

I believe I reviewed those before but must have completely missed it, apologies.

Reply:
All changed in new pushed commit.